docs/api/app.md

```diff
@@ -616,7 +617,7 @@ Returns `string` - The current application directory.
     directory.
   * `temp` Temporary directory.
   * `exe` The current executable file.
-  * `module` The `libchromiumcontent` library.
+  * `module` The location of the Chromium module. By default this is synonymous with `exe`.
```

Comment:
This is only tangentially related to the other changes, but `libchromiumcontent` hasn't existed for a long time, so it seemed worth updating these docs while I'm here.